includes/ucf-email-posttype.php

```diff
@@ -0,0 +1,91 @@
+		 * @author RJ Bruneel
+		 * @since 1.0.0
+		 **/
+		public static function register() {
+			$singular = apply_filters( self::$text_domain . '_singular_label', 'Email' );
+			$plural = apply_filters( self::$text_domain . '_plural_label', 'Emails' );
+			register_post_type( 'ucf-email', self::args( $singular, $plural ) );
+		}
+
+
+		/**
+		 * Returns an array of labels for the custom post type.
+		 * @author RJ Bruneel
+		 * @since 1.0.0
+		 * @param $singular string | The singular form for the CPT labels.
+		 * @param $plural string | The plural form for the CPT labels.
+		 * @return Array
+		 **/
+		public static function labels( $singular, $plural ) {
+			return array(
+				'name'                  => _x( $plural, 'Post Type General Name', self::$text_domain ),
+				'singular_name'         => _x( $singular, 'Post Type Singular Name', self::$text_domain ),
+				'menu_name'             => __( $plural, self::$text_domain ),
+				'name_admin_bar'        => __( $singular, self::$text_domain ),
+				'archives'              => __( $plural . ' Archives', self::$text_domain ),
+				'parent_item_colon'     => __( 'Parent ' . $singular . ':', self::$text_domain ),
+				'all_items'             => __( 'All ' . $plural, self::$text_domain ),
+				'add_new_item'          => __( 'Add New ' . $singular, self::$text_domain ),
+				'add_new'               => __( 'Add New', self::$text_domain ),
+				'new_item'              => __( 'New ' . $singular, self::$text_domain ),
+				'edit_item'             => __( 'Edit ' . $singular, self::$text_domain ),
+				'update_item'           => __( 'Update ' . $singular, self::$text_domain ),
+				'view_item'             => __( 'View ' . $singular, self::$text_domain ),
+				'search_items'          => __( 'Search ' . $plural, self::$text_domain ),
+				'not_found'             => __( 'Not found', self::$text_domain ),
+				'not_found_in_trash'    => __( 'Not found in Trash', self::$text_domain ),
+				'insert_into_item'      => __( 'Insert into ' . $singular, self::$text_domain ),
+				'uploaded_to_this_item' => __( 'Uploaded to this ' . $singular, self::$text_domain ),
+				'items_list'            => __( $plural . ' list', self::$text_domain ),
+				'items_list_navigation' => __( $plural . ' list navigation', self::$text_domain ),
+				'filter_items_list'     => __( 'Filter ' . $plural . ' list', self::$text_domain ),
+			);
+		}
+
+
+		/**
+		 * Returns the arguments for registering the custom post type.
+		 * @author RJ Bruneel
+		 * @since 1.0.0
+		 * @param $singular string | The singular form for the CPT labels.
+		 * @param $plural string | The plural form for the CPT labels.
+		 * @return Array
+		 **/
+		public static function args( $singular, $plural ) {
+			$args = array(
+				'label'                 => __( $plural, self::$text_domain ),
+				'description'           => __( $plural, self::$text_domain ),
+				'labels'                => self::labels( $singular, $plural, self::$text_domain ),
```

Comment:
You pass in the `self::$text_domain` variable, but there's no parameter available to accept it on the labels function.